packages/assets-controllers/src/selectors/token-selectors.test.ts

```diff
@@ -918,5 +921,108 @@ describe('token-selectors', () => {
 
       expect(result).toStrictEqual(expectedMockResult);
     });
+
+    const arrangeTronState = () => {
```

Comment:
I really hate this test suite...

Like sure this uses realistic state data, but MAN its ridiculously difficult to add new tests since you need to understand the WHOLE STATE.

It would be much much nicer if we can:
1. Create JSON objects for each state we want to test. We have a fixtures folder that we can shove this in.
2. Break up the selector parts into modular atomic functions that we can mock and test in isolation.